specs/deneb/p2p-interface.md

```diff
@@ -309,6 +310,12 @@ Clients SHOULD include a sidecar in the response as soon as it passes the gossip
 Clients SHOULD NOT respond with sidecars related to blocks that fail gossip validation rules.
 Clients SHOULD NOT respond with sidecars related to blocks that fail the beacon chain state transition
 
+###### Blobs recovery via local execution layer client
+In addition to `BlobSidecarsByRoot` requests, recent blobs recovery MAY also be done by querying the Execution Layer (i.e. via `engine_getBlobsV1`)
+Implementers are encouraged to leverage this method to increase the likelihood of incorporating and attesting to the last block when its proposer is not able to publish blobs on time.
+
+Clients MUST publish the corresponding `blob_sidecar` on the `blob_sidecar_{subnet_id}` subnet whenever it successfully recovers blobs via local execution layer client. In such case, clients MUST also consider the `blob_sidecar` as "seen" with regards to blob subnet gossip rules.
```

Comment:
When you publishing a message to gossip it gets included in the seen cache right? Why is it necessary to add this sentence here?

> clients MUST also consider the `blob_sidecar` as "seen" with regards to blob subnet gossip rules

Reply:
I'm not referring to libp2p "seen" cache here but rather the equivocation tracking tuple of the gossip rule. I'll revisit it

Reply:
Thinking about adding `(i.e "the sidecar is the first sidecar for the tuple ..." rule)` but doesn't look good to me. Any suggestion on how to improve? Assuming you are ok with the semantics.

Reply:
maybe something like:

```
Clients MUST also update their data structures as if the `blob_sidcar` had been received via gossip (ie updating the anti-equivocation caches)
```

Reply:
Thanks! The clarification sounds better to me